src/libcadet/model/MultiChannelTransportModel.cpp

```diff
@@ -408,12 +409,22 @@ bool MultiChannelTransportModel::configureModelDiscretization(IParameterProvider
 			paramProvider.popScope();
 	}
 
+	
+	clearExchangeModels();
+
+	_exchange.push_back(nullptr);
+
+	_exchange[0] = helper.createExchangeModel("LINEAR_EX");
```

Comment:
Here we should probably ask the parameter provider if there is any exchange model provided and if not default to the linear exchange?